### PR TITLE
Update output checking docs

### PR DESCRIPTION
**Pull request by LFISHER7**
This makes various changes to the output checking page of the docs following feedback.

### CODE REVIEW

File: docs/releasing-files.md [hunk 1]
```diff
@@ -4,42 +4,36 @@
...
+* **Safe data** - what is the potential for individuals to be identified in the data?
+* **Safe settings** - are there technical controls on access to the data?
 * **Safe outputs** - is there any residual risk in outputs being released from the environment?
 
-All data in OpenSAFELY is pseudonymised and all projects and researchers in OpenSAFELY go through an approval process before having any access to the dataset. This ensures the “safe data”, “safe projects” and “safe people” elements of the fives safes framework are met. The application of **disclosure controls** and **output-checking** is part of the “safe outputs” dimension.
+All data in OpenSAFELY is pseudonymised and all projects and researchers in OpenSAFELY go through an approval process before having any access to the dataset. This ensures the “safe data”, “safe projects” and “safe people” elements of the fives safes framework are met. The OpenSAFELY platform ensures that handling risk (“safe settings”) is minimsed. The application of **disclosure controls** and **output-checking** is part of the “safe outputs” dimension.
```
Comment (LisaHopcroft) [inline docs/releasing-files.md:12]: Typo: minimsed, though I'm not convinced that this sentence is clear.
The technical controls implemented as part of the OpenSAFELY platform ensures that “safe settings” are in place and that risk is minimsed in this respect.

File: docs/releasing-files.md [hunk 2]
```diff
@@ -87,18 +81,27 @@ When applying disclosure controls to your outputs, you should consider the poten
...
-*   Counts of zero can be retained.
+*   Redacting counts <=5 in frequency tables. Row and column totals should be recalculated after you have redacted the cell values, to ensure that the redacted values can not be inferred from the totals.
+*   Redacting summaries of numeric variables (eg mean values) describing 5 or fewer patients.
+*   Redacting maximum or minimum values. These often relate to one or two individuals (eg ‘the oldest patient is 103’) and so should be avoided. In some cases the maximum and minima are informative about individuals (‘the target population was schoolchildren from 11 to 16’); these broad sample characteristics are okay.
+*   Redacting graphical figures whose underlying values describe 5 or fewer patients. Figures which include print-outs of patient counts (such as KM plots) should be checked and redacted. Underlying data for plots should be checked - do not rely upon ‘it’s too small to read’ as a justification for having low numbers. These underlying counts should be provided when requesting the release of any figures.
```
Comment (LisaHopcroft) [inline docs/releasing-files.md:91]: Expand KM to Kaplan-Meier (presumably?)

File: docs/releasing-files.md (hunk 1 above)
Comment (LFISHER7) [reply docs/releasing-files.md]: Thanks, I've used your suggestion

**Review by LisaHopcroft — CHANGES_REQUESTED** [review]
Happy to approve once the stray text on line 28 has been removed!

**Review by LisaHopcroft — APPROVED** [review]

**Pull request merged**